geode-core/src/distributedTest/java/org/apache/geode/internal/cache/tier/sockets/InstantiatorPropagationDUnitTest.java

```diff
@@ -424,23 +241,24 @@ public static void startServer() {
    * instantiators get propagated to client when client gets connected.
    */
   @Test
-  public void testServerUpFirstClientLater() throws Exception {
+  public void testServerUpFirstClientLater() {
     PORT1 = initServerCache(server1);
 
     unregisterInstantiatorsInAllVMs();
 
-    Wait.pause(3000);
+    GeodeAwaitility.await().atMost(3, TimeUnit.SECONDS)
+        .untilAsserted(this::assertAllInstantiatorsHaveBeenUnregisteredInAllVMs);
 
-    server1.invoke(InstantiatorPropagationDUnitTest::registerTestObject1);
-    server1.invoke(InstantiatorPropagationDUnitTest::registerTestObject2);
+    server1
+        .invoke(() -> InstantiatorPropagationDUnitTest.registerTestObject("TestObject1", -100123));
+    server1
+        .invoke(() -> InstantiatorPropagationDUnitTest.registerTestObject("TestObject2", -100122));
 
     server1.invoke(() -> InstantiatorPropagationDUnitTest.verifyInstantiators(2));
 
     client1.invoke(() -> InstantiatorPropagationDUnitTest
         .createClientCache(NetworkUtils.getServerHostName(server1.getHost()), PORT1));
 
-    // // wait for client2 to come online
-    Wait.pause(3000);
```

Comment:
Since you removed most of the deprecated Wait.pause() calls from this file, maybe the last one (on line 295) could be removed too

Reply:
Removed this pause